WordPress/wporg-main-2022
# Block: Add "Random Headings" block

Fixes #101, See #107. This adds a new block, "Random Headings," which renders a random heading from our set of 3 on every page load.

Since these just render static headings, we don't have the accessibility concerns of #107.

I did copy over the column width changes so that at the default desktop size (>1000), the heading wraps before "your".

### Screenshots

| Publish | Grow | Flex |
|---------|------|------|
| ![publish-desktop](https://user-images.githubusercontent.com/541093/187743414-95cfdea6-c925-4248-a32d-9e9d93f327ca.png) | ![grow-desktop](https://user-images.githubusercontent.com/541093/187743412-1165cee6-e7ed-4142-a62e-76125f2576da.png) | ![flex-desktop](https://user-images.githubusercontent.com/541093/187743406-eab26747-09cc-497f-a40c-a985d499fbf8.png) |
| ![publish-mobile](https://user-images.githubusercontent.com/541093/187743415-cfcaabcb-4d3b-4f78-957c-db63408230e1.png) | ![grow-mobile](https://user-images.githubusercontent.com/541093/187743413-9d359619-77c5-4156-9a73-f2086e90d948.png) | ![flex-mobile](https://user-images.githubusercontent.com/541093/187743410-3014e2e2-e119-4fcb-ba8b-245e3bc54673.png) |

### How to test the changes in this Pull Request:

1. Build this branch
2. Open the font page of the site — the block has been replaced in the pattern
3. Reload a few times, the heading should change
